Packages/Sites/Neos.NeosIo/Resources/Private/TypoScript/NodeTypes/Reference.List.ts2

```diff
@@ -45,4 +50,14 @@ prototype(Neos.NeosIo:Reference.List.Item) < prototype(TYPO3.TypoScript:Template
 		}
 	}
 	projectType = ${q(q(node).property('projectType')).property('title')}
+
+	@cache {
```

Comment:
This will create a cache entry for every single show case, does that make sense?
IMO it would be enough to have the basic results (everything except fulltext search) cached instead!?